section3/reflection.md

```diff
@@ -1,17 +1,29 @@
 ## Section 3 Reflection
 
 1. What are two points from the Growth Mindset article and/or video that either resonated with you, or were brand new to you?
+* I have seen the SMART goals before, and I have found them helpful when lesson planning for others. I could see them being helpful here.
+* I liked the idea that if I am not encountering challenge, then I am probably not solving any problems.
 
 1. In which ways do you currently demonstrate a Growth Mindset? In which ways do you _not_?
+I am currently very intimidated by challenge in this format, and tend to shut down--the opposite of a growth mindset. On the other hand, I have stuck it out this far.
```

Comment:
You absolutely have stuck it out! It can be daunting and tough, but the point is to learn, grow, AND figure out how you personally learn best. When Matt graduates from Turing he will walk away with the knowledge that he can learn anything and he learns best by _____________________

How can we help you feel less intimidated?